templates/s3_redundant/s3_redundant.py

```diff
@@ -254,3 +264,7 @@ def get_active_ip(self):
 
     def get_passive_ip(self):
         return self._passive_s3().data['mgmtNic']['ip']
+
+    def update_reverse_proxy(self, reverse_proxy):
+        self.data['reverseProxy'] = reverse_proxy
```

Comment:
when you link a reverse_proxy and the s3 service is already installed and running, we need to update the backed servers of the reverse proxy right away, no ? 
So the 3bot create the s3, then the reverse proxy, link the 2 and done, it has his s3 exposed. 
Otherwise there is an extra step where the 3bot need to first read the url of the s3 to create the reverse proxy.

Reply:
yea I thought the 3bot will read the url before creating the reverse_proxy. but updating it here makes more sense of course.

Reply:
It can do that too, but I find the user experience better if you just link the 2 and magic happens